### PR TITLE
Add perfect-scrollbar

### PR DESCRIPTION
**Pull request by noraesae**
It's a custom scrollbar library, aiming for a minimalistic but perfect custom scrollbar.

### CODE REVIEW

File: files/perfect-scrollbar/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "perfect-scrollbar",
+  "repo": "noraesae/perfect-scrollbar",
+  "files": {
+    "include": ["dist/**/*.js", "dist/**/*.css"]
```
Comment (jimaek) [inline files/perfect-scrollbar/update.json]: This config wont retain the directory names. Both js and css files will be added to root. 
Is that correct?

Comment (noraesae) [inline files/perfect-scrollbar/update.json]: Oops, using `files/basePath` seems proper. Will update now. Thanks!

**Comment by noraesae** [comment]
Updated to use `basePath`.

File: files/perfect-scrollbar/info.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author = "Hyunje Alex Jun"
+github = "https://github.com/noraesae/perfect-scrollbar"
+homepage = "http://noraesae.github.io/perfect-scrollbar/"
+description = "Minimalistic but perfect custom scrollbar plugin"
+mainfile = "index.js"
```
Comment (jimaek) [inline files/perfect-scrollbar/info.ini]: `js/min/perfect-scrollbar.min.js`

**Comment by noraesae** [comment]
Updated `mainfile` to `js/min/perfect-scrollbar.min.js`.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**